test/cluster-py/multi.test.py

```diff
@@ -18,11 +18,11 @@
 def check_connection(con):
     try:
         s = con.space('test')
-        print s.select()
+        print(s.select())
```

Comment:
To be honest, I would glad to see a motivation here too (in the commit message). I guess I understand it: test-run is going to support Python 3 and it may be convenient to run tarantool-python's test using Python 3. But will I remember it after a year?..

I'll not insist, but, please, consider my humble opinion: those things are important, because they saves tons of time in a future.

Reply:
Sure, added explanation to commit message and place here for your convenience:

```
commit cb33676fb9cf830c918ee61e09b532fa4aff60a1
Author: Sergey Bronnikov <sergeyb@tarantool.org>
Date:   Wed Nov 25 11:21:20 2020 +0300

    test: make print calls compatible with Python 3.x
    
    In a Python 3 'print' becomes a function, see [1].
    Patch makes print calls compatible with Python 3.
    
    1. https://docs.python.org/3/whatsnew/3.0.html#print-is-a-function
```

Reply:
I would still guess that you had some problem around running on Python 3, but don't described it, if would read your commit message without any context. However, okay, I don't want to tire you with that.